python/lsst/sdm_schemas/schemas/imsim.yaml

```diff
@@ -6015,7 +6015,608 @@ tables:
     datatype: boolean
     description: True when there was no flux attributed to this object after flux redistribution in the deblender.
     fits:tunit:
-
+      # MultiProFit two-Gaussian PSF fits
+  - name: g_psfModel_TwoGaussian_gauss1_sigma_x
+    "@id": "#Object.g_psfModel_TwoGaussian_gauss1_sigma_x"
+    datatype: double
+    description: Standard deviation of the first Gaussian component (x-axis) in the two-Gaussian PSF model (g-band).
+    fits:tunit: pixel
+  - name: g_psfModel_TwoGaussian_gauss1_sigma_y
+    "@id": "#Object.g_psfModel_TwoGaussian_gauss1_sigma_y"
+    datatype: double
+    description: Standard deviation of the first Gaussian component (y-axis) in the two-Gaussian PSF model (g-band).
+    fits:tunit: pixel
+  - name: g_psfModel_TwoGaussian_gauss1_rho
+    "@id": "#Object.g_psfModel_TwoGaussian_gauss1_rho"
+    datatype: double
+    description: Ellipse rho (correlation coefficient) of the first Gaussian component in the two-Gaussian PSF model (g-band).
+    fits:tunit:
+  - name: g_psfModel_TwoGaussian_gauss1_fluxfrac
+    "@id": "#Object.g_psfModel_TwoGaussian_gauss1_fluxfrac"
+    datatype: double
+    description: Fraction of the total flux (normalized to unity) of the first Gaussian component in the two-Gaussian PSF model (g-band).
+    fits:tunit:
+  - name: g_psfModel_TwoGaussian_gauss2_sigma_x
+    "@id": "#Object.g_psfModel_TwoGaussian_gauss2_sigma_x"
+    datatype: double
+    description: Standard deviation of the second Gaussian component (x-axis) in the two-Gaussian PSF model (g-band).
+    fits:tunit: pixel
+  - name: g_psfModel_TwoGaussian_gauss2_sigma_y
+    "@id": "#Object.g_psfModel_TwoGaussian_gauss2_sigma_y"
+    datatype: double
+    description: Standard deviation of the second Gaussian component (y-axis) in the two-Gaussian PSF model (g-band).
+    fits:tunit: pixel
+  - name: g_psfModel_TwoGaussian_gauss2_rho
+    "@id": "#Object.g_psfModel_TwoGaussian_gauss2_rho"
+    datatype: double
+    description: Ellipse rho (correlation coefficient) of the second Gaussian component in the two-Gaussian PSF model (g-band).
+    fits:tunit:
+  - name: g_psfModel_TwoGaussian_cen_x
+    "@id": "#Object.g_psfModel_TwoGaussian_cen_x"
+    datatype: double
+    description: Centroid (x-axis) of the two-Gaussian PSF model (g-band).
```

Comment:
This is relative to the declared center of the original PSF model?  If so, it might be clearer to call it a `delta_x` or something.

Reply:
No, it's in (multiprofit, not converted to afw/numpy convention) PSF image coordinates, so it's a lot of 17.5 +/- 0.005 values. They're not really useful for anything except as a diagnostic in case it's far off x.5, and I guess you can multiply them by 2 and round to get the PSF stamp size.